definitions/ext-service/definition.yml

```diff
@@ -8,13 +8,25 @@ synthesis:
       encodeIdentifierInGUID: true
       conditions:
         - attribute: service_name
+      tags:
+        telemetry.sdk.name:
+          entityTagName: instrumentation.provider
```

Comment:
we can clean it up later but i don't think we should try to override provider in these. Since they already bring a provider.